schism/keybinds_codes.c

```diff
@@ -124,3 +124,268 @@ static struct {
     {"US_KP_PERIOD",  SDL_SCANCODE_KP_PERIOD},
     {"US_KP_EQUALS",  SDL_SCANCODE_KP_EQUALS}
 };
+
+static struct {
+    const char *name;
+    SDL_Keycode code;
+} string_to_keycode[] = {
+    {"ENTER", SDLK_RETURN},
+    {"ESCAPE", SDLK_ESCAPE},
+    {"BACKSPACE", SDLK_BACKSPACE},
+    {"TAB", SDLK_TAB},
+    {"SPACE", SDLK_SPACE},
+    {"EXCLAIM", SDLK_EXCLAIM},
```

Comment:
These are represented as raw ASCII (e.g. `SDLK_EXCLAIM` == `'!'`) so these are a bit redundant...

Reply:
Sure, I just wanted to make sure I didn't miss anything so I just copied the entire list of keycodes and put it in the list.